aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move

```diff
@@ -12,12 +12,6 @@
 /// at the moment.**
 module aptos_framework::aggregator_v2 {
 
-    /// The value of aggregator overflows. Raised by native code.
```

Comment:
Cannot comment on above, there is a module description above, maybe this can be dropped as well for now, since it is a WIP anyway